httpd/handler.go

```diff
@@ -164,7 +164,7 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *influ
 	for {
 		if err := dec.Decode(&br); err != nil {
 			if err.Error() == "EOF" {
-				w.WriteHeader(http.StatusOK)
+				w.WriteHeader(http.StatusCreated)
 				return
```

Comment:
This is the write endpoint, so we actually "created" a entry into our database.  A series/measurement/value was "created" at this point.